### DISCUSSION TITLE
文章无法打开，与日期型的固定链接存在冲突

### DISCUSSION THREAD

**Discussion by bill5522** [Tokinx/Wing#51] [讨论]
这是我自定义的文章固定链接规则：/%year%/%monthnum%%day%%hour%%minute%.html；例：xxx.com/2022/10081715.html
使用主题后无法打开文章，似乎变成了日期筛选页，规则里添加%post_id%可解决。

ps：这个问题排查了好久才找到原因，给后来者标记下

**Comment by Tokinx** [comment]
我使用了官方主题测试发现，这不是一个bug，文章必须要指定一个唯一标识（postid、postname），参考wp后台给的所有示例，均包含postid或postname必要参数。
